new/MDDL002.xml

```diff
@@ -0,0 +1,445 @@
+                            </msItem>
+                            <msItem xml:id="ms_i22">
+                                <locus from="178rc" to="181va"/> 
+                                <title type="complete" ref="LIT3324Severi"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i23">
+                                <locus from="181va" to="181vb"/> 
+                                <title type="complete" ref="LIT3325Gregory"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            <msItem xml:id="ms_i24">
+                                <locus from="181vc" to="183ra"/> 
+                                <title type="complete" ref="LIT3326Melchiz1"/>
+                                <textLang mainLang="gez"/>
+                            </msItem>
+                            
+                        
+                        <msItem xml:id="ms_i27">
+                            <locus from="183rb" to="184vb"/> 
+                            <title type="complete" ref="LIT3327Melchiz2"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i28">
+                            <locus from="184vb" to="185rb"/> 
+                            <title type="complete" ref="LIT3332Melchiz"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i29">
+                            <locus from="185rb" to="185rc"/> 
+                            <title type="complete" ref="LIT3334Ontheb"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i30">
+                            <locus from="185rc" to="186rb"/> 
+                            <title type="complete" ref="LIT3335Antihe"/>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        </msContents>
+                    
+                         <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="186">186</measure>
+                                    <measure unit="page" type="blank">4</measure>
+                                    <locus from="1r" to="1v"/>
+                                    <locus target="#165v #186v"/>
+                                   <note>Measurements of the manuscript's dimensions were not provided in the manuscript's metadata and images.</note>
```

Comment:
maybe dimensions can be estimated using the paper ruler photographed together with the MS

Reply:
I have now used https://imagemeasurement.online/ applying the scale provided and can give the 

outer measurements
height: 326 mm
width: 286 mm

text area (f. 5r)
height: 205 mm
width: 215 mm